### PR TITLE
Imuf 104

### PR DESCRIPTION
**Pull request by rs2k**

### CODE REVIEW

File: src/main/sensors/gyro.c [hunk 1]
```diff
@@ -189,14 +190,15 @@ PG_RESET_TEMPLATE(gyroConfig_t, gyroConfig,
     .gyro_soft_notch_cutoff_1 = 0,
     .gyro_soft_notch_hz_2 = 0,
     .gyro_soft_notch_cutoff_2 = 0,
-    .checkOverflow = GYRO_OVERFLOW_CHECK_ALL_AXES,
+    .checkOverflow = GYRO_OVERFLOW_CHECK_YAW,
     .imuf_mode = 32,
+    .imuf_rate = IMUF_RATE_32K,
     .imuf_pitch_q = HELIO_PROFILE_PITCH_Q,
-    .imuf_pitch_r = 88,
+    .imuf_pitch_w = HELIO_PROFILE_PITCH_W,
```
Comment (apocolipse) [inline src/main/sensors/gyro.c]: These should not take from a specific target for future target compatibility

File: src/main/sensors/gyro.h [hunk 2]
```diff
@@ -56,6 +56,21 @@ typedef enum {
...
 
+#if defined(USE_GYRO_IMUF9001)
+typedef enum {
+    IMUF_RATE_32K = 0,
+    IMUF_RATE_16K = 1,
+    IMUF_RATE_8K = 2,
+    IMUF_RATE_4K = 3,
+    IMUF_RATE_2K = 4,
+    IMUF_RATE_1K = 5,
+    IMUF_RATE_05K = 6,
```
Comment (apocolipse) [inline src/main/sensors/gyro.h]: Is this appropriate naming convention given there's no decimal?  why not just `IMUF_RATE_500`,  `IMUF_RATE_250`, `IMUF_RATE_125`,  etc.

File: src/main/drivers/pwm_output.c [hunk 3]
```diff
@@ -48,7 +48,7 @@ static uint16_t freqBeep = 0;
 static bool pwmMotorsEnabled = false;
 static bool isDshot = false;
 #ifdef USE_DSHOT_DMAR
-#ifdef FORCE_DSHOT_DMAR
+#ifdef ENABLE_DSHOT_DMAR
```
Comment (kidBrazil) [inline src/main/drivers/pwm_output.c]: Why the change here?

File: src/main/sensors/gyro.c (hunk 1 above)
Comment (orneryd) [reply src/main/sensors/gyro.c]: I will change it to IMUF and ensure it's defined in the driver header file

File: src/main/drivers/pwm_output.c (hunk 3 above)
Comment (orneryd) [reply src/main/drivers/pwm_output.c]: the rest changed upstream so we changed it to match

File: src/main/sensors/gyro.h (hunk 2 above)
Comment (orneryd) [reply src/main/sensors/gyro.h]: I'm going to also keep them in sync automatically with the dropdown

Comment (orneryd) [reply src/main/sensors/gyro.h]: I also removed the lower denominations since it doesn't make sense to and I can constrain the value when set to be 5 (1k) or less based on the gyro denom.
